Jobba.Core/Implementations/DefaultJobScheduler.cs

```diff
@@ -135,11 +135,10 @@ public async Task<JobInfo<TJobParams, TJobState>> DoScheduleJobAsync<TJobParams,
 
         var jobInfo = await UpdateAttemptsOrCreateJobAsync(request, cancellationToken);
         await _jobStore.SetJobStatusAsync(jobId, JobStatus.Enqueued, DateTimeOffset.UtcNow, cancellationToken);
-        var token = _jobCancellationTokenStore.CreateJobCancellationToken(jobId, cancellationToken);
```

Comment:
yes, we still get that token but inside the jobrunner since it is not used anywhere else in this method. I created the JobRunner because I needed to retrieve more services inside the scope to fix a bug, so it made sense to me to create the scope and then resolve one service whose dependencies would all be resolved within that scope